packages/flutter/test/widgets/routes_test.dart

```diff
@@ -503,6 +504,52 @@ void main() {
       verifyNoMoreInteractions(pageRouteAware);
     });
   });
+
+  group('ModalRoute', () {
+    testWidgets('should resume focus when the next route is popped', (WidgetTester tester) async {
+      // TODO(tongmu): It's currently using PageRoute (from MaterialApp)
```

Comment:
Can this be considered a `ModalRoute` test if it is using a `PageRoute` workaround?

Reply:
Yeah good point. I think the idea is that the code that it's testing is in ModalRoute. It's still a "ModalRoute test", it's just not good enough.